backend/tests/test_batches.py

```
@@ -0,0 +1,298 @@
+        created = await batches.create_batch(
+            _request("one.md", "two.md"), Response(), None, db, owner
+        )
+        item_id = created["items"][0]["id"]
+        item = next(
+            document
+            for document in db.batch_items.documents
+            if document["id"] == item_id
+        )
+        item.update(
+            state="running",
+            claim_token="abandoned-claim",
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "claim_token"

(S106)